src/ddscxx/tests/Listener.cpp

```diff
@@ -308,8 +309,7 @@ class Listener : public ::testing::Test
     }
 
     void SetUp() {
-        char buf[32];
-        ddsrt_pid_t pid = ddsrt_getpid();
+        char name[32];
```

Comment:
32 characters is suffcient?

Not that I am really complaining, we have to give the string some size, just that this introduces a possibility of failure which can be triggered by giving a too large string.